src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -61,37 +70,55 @@
    :padding-right  9})
 
 (def separator
-  {:background-color "rgb(200, 199, 204)"
-   :height           0.5})
+  {:background-color "rgb(238, 241, 245)"
+   :height           2
+   :margin-top       2
+   :margin-bottom    2})
 
 ;; Popular list item
 
 (def popular-list-container
   {:flex             1
    :background-color :white
    :padding-top      18
-   :padding-left     16})
+   :padding-left     16
+   })
 
 (def popular-list-item
-  {:flex-direction :row
+  {:flex-direction "column"
```

Comment:
is there a reason this i "column" vs :column? Considering we had :row before

Reply:
I agree, let's just make sure what's the "canonical" form. I wasn't sure.

Reply:
Not sure either, I'd just keep whatever form seems dominant - a grep for `flex-direction` shows 80-90% use keyword so let's stick with that

Reply:
Keywords are better in Clojure